raisinbread/README.md

```diff
@@ -29,8 +29,17 @@ that is not the case, replace all `mysql` commands below by the necessary values
 `mysql -u user -p -h host database_name`*
 
 ##### Sourcing SQL
-In order to be able to use the RaisinBread dataset, the LORIS SQL schema needs to be 
-sourced, followed by the different instrument schemas and finally the actual RB data. 
+If the database being used is already populated and contains any tables or data, the following
+command must be used in the main LORIS root directory:
```

Comment:
I would recommend rephrasing this sentence to indicate very clearly that the commands below will effectively erase all data in the database and to make sure a backup is available if the existing data is immportant.

Reply:
I will add that information now. Thank you :)